src/SvgIcon/SvgIcon.js

```diff
@@ -39,6 +39,7 @@ export default function SvgIcon(props, context) {
     <svg
       className={className}
       viewBox={viewBox}
+      aria-hidden="true"
```

Comment:
Should we instead be adding the icon name as its title? What if this was in an Icon button? (Or is it up to the user to meaningfully label the button?)

https://www.w3.org/TR/SVG-access/#Equivalent